bakery/src/scripts/link_single.py

```diff
@@ -153,7 +155,18 @@ def transform_links(
         link = node.attrib["href"]
 
         target_module_uuid = get_target_uuid(link)
-        canonical_book_uuid = canonical_map[target_module_uuid]
+        canonical_book_uuid = canonical_map.get(target_module_uuid)
+
+        if ((not canonical_book_uuid == source_book_uuid) and mock_otherbook):
+            # If the canonical book UUID doesn't equal the current book (which
+            # includes if the lookup returned None) and we're mocking otherbook
+            # links, go ahead and insert the mock.
+            node.attrib["href"] = "mock-inter-book-link"
```

Comment:
Should we add a mock `data-book-uuid` value to this as well, since that's the only thing that indicates something is an inter-book link?

Reply:
[Yep](https://github.com/openstax/output-producer-service/pull/304/commits/36972b4e110bf4cccdc3ba2dd541595cc531e229)!